jenkins/release.jenkinsFile

```diff
@@ -0,0 +1,20 @@
+lib = library(identifier: 'jenkins@5.9.1', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+standardReleasePipelineWithGenericTrigger(
+    overrideDockerImage: 'opensearchstaging/ci-runner:release-centos7-clients-v4',
```

Comment:
No need to override, the 5.9.1 version has this by default: https://github.com/opensearch-project/opensearch-build-libraries/blob/5.9.1/vars/standardReleasePipelineWithGenericTrigger.groovy#L31

Reply:
Updated.